controllers/function_mesh.go

```diff
@@ -389,6 +503,33 @@ func (r *FunctionMeshReconciler) UpdateFunctionMesh(ctx context.Context, req ctr
 		}
 	}
 
+	if len(mesh.Spec.GenericResources) != len(mesh.Status.GenericResourceConditions) {
+		for resourceName, resourceCondition := range mesh.Status.GenericResourceConditions {
+			if resourceCondition.Condition == v1alpha1.Orphaned {
+				// clean up the orphaned generic resource
```

Comment:
Shouldn't these resources have deletion timestamps as well if this is the case?

Reply:
no, these resources are not refering to a k8s resources directly, but just a condition struct in function-mesh